_episodes/02-pulling-images.md

```diff
@@ -10,8 +10,8 @@ objectives:
 - "List local images"
 - "Introduce image tags"
 keypoints:
-- "Pull images with `docker pull <image-id>`"
-- "List all images on the computer and other information with `docker images`"
+- "Pull images with `podman pull <image-id>`"
+- "List all images on the computer and other information with `podman images`"
 - "Image tags distinguish releases or version and are appended to the image name with a colon"
```

Comment:
The key points don't seem to appear on the page...?

Reply:
They are shown at the bottom 
<img width="1237" alt="image" src="https://github.com/hsf-training/hsf-training-docker/assets/2147367/a453197a-c118-4162-90b6-60c37d51957b">

Reply:
oh, yes, so sorry